### PR TITLE
Include heading anchors for website

### PR DESCRIPTION
**Pull request by timotheeguerin**

### CODE REVIEW

File: packages/website/src/docs/language-basics/intersections.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Intersections describe a type that must include all the intersection's constitue
 alias Dog = Animal & Pet;
 ```
 
-An intersection is equivalent to [spreading]({%doc "models"%}#spread) both types.
+An intersection is equivalent to [spreading]({%doc "models", "#spread"%}) both types.
```
Comment (timotheeguerin) [inline packages/website/src/docs/language-basics/intersections.md:14]: Is this change worth it or the way it was done before is fine? (Both work right now)

Comment (nguerrera) [reply packages/website/src/docs/language-basics/intersections.md:14]: This is just a syntactic issue of how these refrences are made and they both come out the same. If so, I have no opinion, lol.

**Review by nguerrera — APPROVED** [review]

File: packages/website/src/docs/language-basics/intersections.md (hunk 1 above)
Comment (timotheeguerin) [reply packages/website/src/docs/language-basics/intersections.md:14]: yeah basically same result

**Review by bterlson — APPROVED** [review]

**Pull request merged**